dothething/dothething/Views/ProfileView.swift

```diff
@@ -264,7 +264,7 @@ extension ProfileView {
                             // save url to clips array
                             DispatchQueue.main.async {
                                 let thumbnail = Thinger.getThumbnail(url: tempFileUrl, degreesToRotate: self.videoDegreesToRotate)
-                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip))
+                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip, showCode: true))
```

Comment:
The change to set `showCode` to `true` for newly added clips in the `ProfileViewModel` is a logical decision, as it aligns with the overall design and functionality of the app. Users will expect to see the code metadata for clips displayed in the profile view, just as they do in the main clips view.